skype/skype-ps/skype/Disable-CsOnlineDialInConferencingUser.md

```diff
@@ -13,6 +13,10 @@ ms.reviewer:
 # Disable-CsOnlineDialInConferencingUser
 
 ## SYNOPSIS
+
+> [!NOTE]
+> This cmdlet will be deprecated on December 31, 2021. To disable Audio Conferencing for a user, please unassign the Audio Conferencing license from the user or disable the Audio Conferencing component from the license that is assigned to the user. For additional information, see [Assign licenses to users](https://docs.microsoft.com/en-us/microsoft-365/admin/manage/assign-licenses-to-users).
```

Comment:
```suggestion
> This cmdlet will be deprecated on December 31, 2021. To disable Audio Conferencing for a user, please unassign the Audio Conferencing license from the user or disable the Audio Conferencing component from the license that is assigned to the user. For additional information, see [Assign licenses to users](https://docs.microsoft.com/microsoft-365/admin/manage/assign-licenses-to-users).
```

Reply:
I didn't commit this change because I changed the link to a relative link - that included removing "en-us"....